src/macros.rst

```diff
@@ -1044,37 +1044,33 @@ environment.
 Hygiene is categorized as follows:
 
 * :dp:`fls_dz2mvodl818d`
-  *Definition site hygiene*, which resolves to the ``MacroDeclaration``
+  :dt:`Definition site hygiene`, which resolves to the ``MacroDeclaration``
   site. ``Identifier``\ s with definition site hygiene cannot reference
```

Comment:
Shouldn't `Identifier` actually be `Path`? Wither way, it is missing the :s: tag.

Also, `definition site hygiene` should reference the term.